include/mbedtls/aes.h

```diff
@@ -308,13 +308,22 @@ int mbedtls_aes_crypt_cfb8( mbedtls_aes_context *ctx,
  *        The OFB operation is identical for encryption or decryption, therefore
  *        no operation mode needs to be specified.
  *
- * \note  Upon exit, the content of the IV is updated so that you can
- *        call the same function again on the next
- *        block(s) of data and get the same result as if it was
- *        encrypted in one call. This allows a "streaming" usage.
- *        If you need to retain the contents of the
- *        IV, you must either save it manually or use the cipher
- *        module instead.
+ * \note  Upon exit, the content of iv, the Initialisation Vector, is updated
+ *        so that you can call the same function again on the next block(s) of
+ *        data and get the same result as if it was encrypted in one call. This
+ *        allows a "streaming" usage, by initialising iv_off to 0 before the
+ *        first call, and preserving its value between calls.
+ *
+ *        For block by block usage, (or non-streaming use), the iv should be
```

Comment:
I don't think it's a good idea to use the word "block" here - people might interpret it as an AES block (16 bytes) and I don't think that's what sentence is about.

Reply:
Oh, good point! Bit careless use of language.

Done.